docs/glossary.md

```diff
@@ -0,0 +1,39 @@
+Glossary
+========
+   Node: A physical machine that is running one replica per protocol instance. The nodes are known to each other.
```

Comment:
Please use bullet points for better readability

Reply:
Done